README.md

```diff
@@ -4,36 +4,148 @@ Fast, cross-platform mail server.
 
 Inspired from [Caddy](https://github.com/mholt/caddy).
 
-## Features
-
-* Easy configuration with Maddyfile (same as Caddyfile)
-* Automatic TLS and DKIM support
-* Transparent PGP support
-* Runs anywhere with no external dependencies (not even libc)
-
-## Usage
+## Installation
 
 ```shell
-go get github.com/emersion/maddy/...
+go get github.com/emersion/maddy/cmd/maddy
 ```
 
-Create a file named `Maddyfile`:
+Build tags:
+* `nosqlite3`
+  Disable SQLite3 support in go-sqlmail (enabled by default). Saves around 9
+  MiB of binary size.
+* `mysql`
+  Include support for MySQL driver in go-sqlmail.
+* `postgresql`
+  Include support for PostgreSQL driver in go-sqlmail.
+
+## Configuration
+
+### Syntax
+
+Maddy uses configuration format similar (but not the same!) to Caddy's
+Caddyfile.  You may want to read [Caddyfile page from Caddy docs](https://caddyserver.com/docs/caddyfile).
+
+Notable differences from Caddy's format:
+* Directives can't span multiple lines
+* Environmental variables are not expanded
```

Comment:
I'm used to say "environment variables", is this a typo or is it just me?

Reply:
Not sure how I got used to typing "environmental", but yes, this is a typo.